src/main/java/sirius/biz/importer/format/FieldDefinition.java

```diff
@@ -38,6 +38,7 @@
     protected Supplier<String> label;
     protected Set<String> aliases = new HashSet<>();
     protected List<ValueCheck> checks = new ArrayList<>();
+    protected boolean includeIdentifierAndValueInErrorMessage;
```

Comment:
That is a very long variable name :-)
perhaps `appendContextInErrorMessage` ?

Reply:
Has been renamed 👍